_tests/browser_tests/.eslintrc

```diff
@@ -0,0 +1,10 @@
+env:
+  node: true
```

Comment:
Sweet. I'll leave it to match the settings in the unit tests and keep it in case the main eslintrc file changes.